Src/WindowsServer/WindowsServer.NetCore/WindowsServer.NetCore.csproj

```diff
@@ -0,0 +1,50 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props" Condition="Exists('..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Signing.props'))\Signing.props" Condition="'$(DoNotSign)' == ''" />
+
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props'))" />
+    <Error Condition="!Exists('..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.targets'))" />
+  </Target>
+
+  <Import Project="..\WindowsServer.Shared\WindowsServer.Shared.projitems" Label="Shared" />
+
+  <Import Project="..\..\Common\Common.projitems" Label="Shared" />
+
+  <Import Project="..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.targets" Condition="Exists('..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.targets')" />
+
+  <PropertyGroup>
+    <Copyright>Copyright ? Microsoft. All Rights Reserved.</Copyright>
+    <VersionPrefix>2.6.0-beta2</VersionPrefix>
+    <Authors>Microsoft</Authors>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DefineConstants>$(DefineConstants);NETCORE</DefineConstants>
```

Comment:
Careful. You're conflating a runtime (.NET Core) concept with a standard library (NETSTANDARD1_6) target framework. These things are not equivalent. A netstandard library can be consumed by many different runtimes (.NET Framework, Mono, CoreCLR). This may lead to confusion later on. Note that the project system automatically defines NETSTANDARD1_6 for you if the TargetFramework is netstandard1.6 and there are similar defines for other target frameworks. If you need a preprocessor constant that covers both NETSTANDARD1_3 and NETSTANDARD1_6, I would recommend NETSTANDARD.

Reply:
Following on from this, I would rename the folder and project "WindowsServer.NetStandard16" instead of "WindowsServer.NetCore"

Probably not in this PR, but we should then try to consolidate all the targets (net40, net45, netstandard16) into a single .csproj with multiple target frameworks. Like we did in the base SDK.

Reply:
I see, thanks for calling this out. I imagine this is a result of my blind copying some of the other NetCore projects - test projects specifically that are actually netcore1.0 apps.

Reply:
Done. I will leave the multi-targeted project to later, but I agree we need to get that done as well. I opened issue #863.

Reply:
On the subject of renaming the projects to NetStandard instead of NetCore - I'll also leave that to the consolidation issue. NetCore was the naming convention for this type of work before I got here (correct or not) and I'm going to call fixing that out of scope for this PR.